### DISCUSSION TITLE
Get a list of IPs by ids

### DISCUSSION THREAD

**Discussion by arnaud-barre** [netbox-community/netbox#12364] [Q&A]
### NetBox version

v3.4.9

### Feature type

Change to existing functionality

### Proposed functionality

It could be great to have APIs to retrieve a list of objects by a list of ids. For exemple, it could be great to have /ipam/ip-addresses/?ids=1234,4567,7895

This could be done for all objects (prefixes, tags, ...) and could avoid to have multiple rest request to retrieve this list of objects.

### Use case

Retrieve a list of objects by a list of ids :

For example : /ipam/ip-addresses/?ids=1234,4567,7895

**Comment by jeremystretch** [comment]
This is already possible. You just need to use this format: `/ipam/ip-addresses/?id=1&id=2&id=3`
